examples/ding-dong-bot.java

```diff
@@ -1,4 +0,0 @@
-/**
```

Comment:
May we keep this file so that a new developer can be easy to find out the main entrance of our Java example, especially a developer who is not very familiar with the Java directory structure?